app/kumactl/cmd/apply/apply_test.go

```diff
@@ -178,6 +182,40 @@ var _ = Describe("kumactl apply", func() {
 		Expect(resource.Meta.GetNamespace()).To(Equal("default"))
 	})
 
+	It("should apply a new Dataplane resource from URL", func() {
+		// setup http server
+		mux := http.NewServeMux()
+		server := httptest.NewServer(mux)
+		port, err := strconv.Atoi(strings.Split(server.Listener.Addr().String(), ":")[1])
+		testurl := fmt.Sprintf("http://localhost:%v/testdata/apply-dataplane.yaml", port)
+		Expect(err).ToNot(HaveOccurred())
+
+		// then
+		mux.Handle("/testdata/", http.StripPrefix("/testdata/", http.FileServer(http.Dir("./testdata"))))
+		Eventually(func() bool {
+			resp, err := http.Get(testurl)
+			if err != nil {
+				return false
+			}
+			Expect(resp.Body.Close()).To(Succeed())
+			return true
+		}).Should(BeTrue())
```

Comment:
By default it's 1s. This should be enough, but let's change this to 5s to be sure that server is started so we can avoid flakiness of the test.
`}, "5s", "100ms").Should(BeTrue())`

Reply:
done